runtime/onert/core/src/compiler/ShapeValidator.h

```diff
@@ -39,8 +39,13 @@ class ShapeValidator : public ir::OperationVisitor
 public:
   ShapeValidator(void) = delete;
   ShapeValidator(const ir::Graph &graph);
+  ShapeValidator(const ShapeValidator &) = delete;
+  ShapeValidator(ShapeValidator &&) = delete;
+  ~ShapeValidator() = default;
 
 public:
+  ShapeValidator &operator=(const ShapeValidator &) = delete;
```

Comment:
An implicitly-declared copy assignment operator for this class is defined as `deleted` because of the member `const ir::Graph &_graph;` (const or reference member), but I explicitly declared `deleted copy assignment operator` because of `The Rule of Five`.